common/predictive-text/worker/models/common.ts

```diff
@@ -0,0 +1,25 @@
+namespace models {
+  export class Common {
```

Comment:
Does this work if you define it as a `namespace` instead of a class? It isn't really a class...

Reply:
I could leave it as a stray function; that is possible in TypeScript.  I'm probably too used to Java, with its "thou shalt put everything in a class."